From f78b725c5eddd10755ef7ec80b9562ea6d647ed6 Mon Sep 17 00:00:00 2001
From: Aleksandr Dolgaryev <adolgarev@gmail.com>
Date: Fri, 16 Sep 2016 11:03:34 +0300
Subject: [PATCH] rest_framework and admin static files serve via uwsgi

---
 uwsgi_dev.ini | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/uwsgi_dev.ini b/uwsgi_dev.ini
index 5882ce9..c5f0a91 100644
--- a/uwsgi_dev.ini
+++ b/uwsgi_dev.ini
@@ -7,4 +7,6 @@ max-requests = 5000
 processes = 2
 touch-reload = /web/uwsgi_dev.ini
 static-map = /=/web/client/build
+static-map = /static/rest_framework=/web/static/rest_framework
+static-map = /static/admin=/web/static/admin
 static-index = index.html
-- 
2.9.3

